tests/NovelRT.Tests/Ecs/CatalogueTest.cpp

```diff
@@ -21,7 +21,7 @@ class CatalogueTest : public testing::Test
         componentCache = ComponentCache(1);
         entityCache = EntityCache(1);
         componentCache.RegisterComponentType<int32_t>(-1, "THROW_AWAY");
-        componentCache.RegisterComponentType<size_t>(-1, "THROW_AWAY_AGAIN");
+        componentCache.RegisterComponentType<size_t>(1, "THROW_AWAY_AGAIN");
```

Comment:
This is gonna break at some point - please change it to numeric limits max or back to -1.